sdk/template/template/README.md

```diff
@@ -1,27 +1,95 @@
-## Azure Template client library for JavaScript
+# Azure Template client library for JavaScript
 
-This template serves as a starting point for JavaScript libraries targeting both Node and the Browser and implemented in TypeScript.
+<!-- NOTE: This README file is a template. Read through it and replace the instructions (keeping an eye out for package names like "@azure/template") with the ones that pertain to your package. For a complete example based on Azure Text Analytics, see README-TEMPLATE.md in this directory. -->
```

Comment:
```suggestion
> NOTE: This README file is a template. Read through it and replace the instructions (keeping an eye out for package names like "@azure/template") with the ones that pertain to your package. For a complete example based on Azure Text Analytics, see README-TEMPLATE.md in this directory.
```

Reply:
This way they show up when rendering the markdown file

Reply:
I wrote them as comments so that they would show up to someone editing the file to create a new package, but not to someone viewing it on GitHub or NPM (we do publish the template on NPM).

Perhaps `README-TEMPLATE` is the wrong name for that file. It should be `README-EXAMPLE`. This README.md is actually the barebones template for our README files.